tests/example_workflows/pyxaid2/example_1_Si/step1/x0.md.in

```diff
@@ -2,7 +2,7 @@
   calculation = 'md',
   dt = 20.67055,
   nstep = 1000,
-  pseudo_dir = '/util/academic/espresso/6.2.1/pseudopotentials/upf_files/',
+  pseudo_dir = '/gpfs/scratch/brendan/PP/',
```

Comment:
same as with submit.slm, but unlike that one, it makes more specific changes - which will not work even for people on CCR (unlike the previous commit), so this should not be included.